README.md

```diff
@@ -3,3 +3,44 @@
 String processing library for Urbit.
 
 Work in progress, feedback and pull requests welcome!
+
+### Installing / Using
+
+1. Copy lib/string.hoon into {pier}/{desk}/lib
+1. Copy app/string-tester.hoon into {pier}/{desk}/app
```

Comment:
Just `cp -r ./* pier/desk/`? I know that currently breaks on the readme, but urbit/arvo#1121 will likely force that to be fixed.

Reply:
Hah. I was actually going to put a plug in for my https://github.com/ngzax/urbit-sync project which will copy it over and ignore the README but I thought that was too self-serving. :)

If you feel strongly I can change it, but I think it's basically the same and doesn't cause an error now.

Reply:
Readme error has been fixed, so we should be good here. (:

Reply:
All right. I'll fix these up and add a new commit today. 😄